app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+
+def create_person_list(people: list) -> list[Person]:
+    new_people_list = []
+
+    for human in people:
+        new_people_list.append(Person(human["name"], human["age"]))
+
+    for human in people:
+        if "wife" in human and human["wife"]:
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        elif "husband" in human and human["husband"]:
```

Comment:
Similarly, use the `dict.get()` method here instead of explicitly checking for the presence of a key. For example: `elif human.get('husband'):`